src/app/page/Signup/SignupScene.jsx

```diff
@@ -125,33 +120,20 @@ function SignupScene(props) {
       break;
     case Tabs.PROFILE:
       Active = (
-        <UserProfilePage handleChange={handleChange} onSubmit={updateUser} values={values} />
+        <UserProfilePage
+          handleChange={handleChange}
+          onSubmit={handleProfileUpdate}
+          values={values}
+        />
       );
       break;
     case Tabs.SUCCESS:
     default:
-      Active = <SignupSuccessPage onClick={() => props.history.push(routes.home)} />;
+      Active = <SignupSuccessPage handleButtonClick={() => props.history.push(routes.home)} />;
       break;
   }
 
-  const showSuccessSnackbar = !errorSnackbarMessage && activeTab === 4;
-  const snackbarOpen = errorSnackbarMessage || showSuccessSnackbar;
-  const snackbarType = showSuccessSnackbar ? "success" : "error";
-  const snackbarMessage = showSuccessSnackbar
-    ? "Your account request has been successfully submitted and is pending approval"
-    : `Error while submitting request. Please try again. ${errorSnackbarMessage}`;
-  return (
-    <>
-      {Active}
-      <Snackbar
-        open={snackbarOpen}
-        handleClose={() => setErrorSnackbarMessage(false)}
-        type={snackbarType}
-        message={snackbarMessage}
-        autoHideDuration={4000}
-      />
-    </>
-  );
+  return <>{Active}</>;
```

Comment:
We won't need the fragment here since there is only one element being returned.

If i recall correctly, since Active is assigned to a component,  you can return `<Active />` or just `Active`